docs/devops-guide/docker.md

```diff
@@ -446,6 +446,11 @@ Variable | Description | Default value
 `DISABLE_HTTPS` | Handle TLS connections outside of this setup | 1
 `ENABLE_HTTP_REDIRECT` | Redirect HTTP traffic to HTTPS (necessary for Let's Encrypt) | 1
 `LOG_LEVEL` | Controls which logs are output from prosody and associated modules | info
+`SENTRY_DSN_JICOFO` | Enables [Sentry error tracking](#error-tracking) for jicofo | none
+`SENTRY_DSN_JVB` | Enables [Sentry error tracking](#error-tracking) for videobridge | none
+`SENTRY_DSN_JIGASI` | Enables [Sentry error tracking](#error-tracking) for jigasi | none
```

Comment:
Link should be the link to the logging page and variables should be named like in the docker repository.